dev/set_matrix.py

```diff
@@ -61,16 +61,16 @@ def create_dev(cls):
 
 class PackageInfo(BaseModel):
     pip_release: str
-    install_dev: t.Optional[str]
+    install_dev: t.Optional[str] = None
```

Comment:
I guess `install_dev: t.Optional[str]` is like `must exist, accepts null` in pydantic V2.